src/main/fc/tuning_sliders.c

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Humming and harring a bit if this is in the right place - I think `src/main/config` is better.

Reply:
I'll move it. Wasn't sure where it should go.